client/ayon_resolve/api/plugin.py

```diff
@@ -746,3 +745,13 @@ def _store_new_instance(self, new_instance):
 
         # Add instance to current context
         self._add_instance_to_context(new_instance)
+
+
+def get_representation_files(representation):
+    anatomy = Anatomy()
```

Comment:
```suggestion
def get_representation_files(project_name, representation):
    anatomy = Anatomy(project_name)
```

Reply:
This is adjusted, thanks for the feedback.